content/cli/commands/cluster.md

```diff
@@ -1,5 +1,266 @@
+NAME: fluvio
+LAST DEPLOYED: Mon Dec 28 13:12:27 2020
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+✅ ok: Kubernetes config is loadable
+✅ ok: Supported helm version is installed
+✅ ok: Fluvio system charts are installed
+✅ ok: Previous fluvio installation not found
+✅ ok: Load balancer is up
+```
+
+To start a cluster locally (as processes on your machine):
+
+```bash
+$ fluvio cluster start --local
+Performing pre-flight checks
+✅ ok: Supported helm version is installed
+✅ ok: Supported kubernetes version is installed
+✅ ok: Kubernetes config is loadable
+✅ ok: Fluvio system charts are installed
+```
+
+## `fluvio cluster delete`
+
+Deletes a Fluvio cluster and all data associated with it. Be careful, this
+cannot be undone.
+
+```
+fluvio-cluster-delete
+Delete a Fluvio cluster from the local machine or Minikube
+
+USAGE:
+    fluvio cluster delete [FLAGS] [OPTIONS]
+
+FLAGS:
+        --no-wait    don't wait for clean up
+        --local      Remove local spu/sc(custom) fluvio installation
+        --sys        Remove fluvio system chart
+    -h, --help       Prints help information
+
+OPTIONS:
+        --namespace <namespace>     [default: default]
+        --name <name>               [default: fluvio]
+```
+
+Example usage:
+
+To uninstall Fluvio from Kubernetes (e.g. Minikube):
+
+```bash
+$ fluvio cluster delete
+```
+
+To uninstall Fluvio from your local machine:
+
+```bash
+$ fluvio cluster delete --local
+```
+
+## `fluvio cluster releases list`
+
+Prints all the published versions of Fluvio that are candidates to
+install on Kubernetes with `fluvio cluster start`.
+
+The Fluvio cluster components are distributed as Helm charts which allow
+them to be installed easily on Kubernetes. This prints all the releases
+that are available.
+
+```
+fluvio-cluster-releases-list
+Show a list of Fluvio release versions
+
+USAGE:
+    fluvio cluster releases list
+
+FLAGS:
+    -h, --help    Prints help information
+```
+
+Example usage:
+
+```bash
+$ fluvio cluster releases list
+VERSION
+0.6.0-latest
+0.6.0-beta.1-latest
+0.6.0-beta.1
+0.6.0-alpha.8-latest
+0.6.0-alpha.7-latest
+...
+```
+
+## `fluvio cluster spu list`
+
+This command shows details about the active SPUs in your cluster.
+It is mostly useful for checking on the status of individual SPUs
+to see whether they are still online, and which addresses they live at.
+
+```
+fluvio-cluster-spu-list
+List all SPUs known by this cluster (managed AND custom)
+
+fluvio cluster spu list [FLAGS] [OPTIONS]
+
+FLAGS:
+        --custom    Whether to list only custom SPUs
+    -h, --help      Prints help information
+
+OPTIONS:
+    -O, --output <type>    Output [default: table]  [possible values: table,
+                           yaml, json]
+```
+
+Example usage:
+
+```bash
+$ fluvio cluster spu list
+ ID    NAME             STATUS  TYPE      RACK  PUBLIC          PRIVATE
+ 5001  custom-spu-5001  Online  "custom"   -    localhost:9010  localhost:9011
+```
```

Comment:
There are a few other commands that need  documentation
* fluvio cluster spu register/unregister
* fluvio cluster spg ...
* fluvio cluster upgrade